internal/integratedservices/services/logging/operator_flow_resource.go

```diff
@@ -25,6 +25,11 @@ import (
 )
 
 func (op IntegratedServiceOperator) createClusterFlowResource(ctx context.Context, managers []outputDefinitionManager, clusterID uint) error {
+	if managers == nil || len(managers) == 0 {
```

Comment:
```suggestion
	if len(managers) == 0 {
```
This check is enough for both cases.